cypress/support/e2e.ts

```diff
@@ -9,24 +9,36 @@ Cypress.on('window:before:load', (win) => {
 })
 
 beforeEach(() => {
-    cy.intercept('POST', '**/decide/*').as('decide')
-    cy.intercept('POST', '**/e/*').as('capture')
-    cy.intercept('POST', '**/ses/*').as('session-recording')
-    cy.intercept('GET', '**/surveys/*').as('surveys')
+    cy.intercept('POST', '/decide/*').as('decide')
```

Comment:
as far as i can tell `'/decide/*'` matches more than `'**/decide/*'` 

from console errors it appears `**/decide` does not match a call that is simple `/decide` (which the browser interprets as `{current base url}/decide`